index.js

```diff
@@ -11,6 +37,25 @@ module.exports = {
     };
   },
 
+  included: function(app) {
+    app.options.autoRun = false;
+    app.options.storeConfigInMeta = false;
+  },
+
+  config: function(/* environment, appConfig */) {
+    // do nothing unless running `ember fastboot` command
+    if (!process.env.EMBER_CLI_FASTBOOT) { return {}; }
+
+    return {
+      EmberENV: {
+        FEATURES: { 'ember-application-visit': true }
+      },
+      APP: {
+        autoboot: false
+      }
```

Comment:
both of these works (they merges correctly as far as I can tell, can add a test too)